masonry/src/doc/01_creating_app.md

```diff
@@ -0,0 +1,238 @@
+# Building a "To-Do List" app
+
```

Comment:
This still needs the newlines around the opening/closing tags, because [the rendering](https://github.com/linebender/xilem/blob/837339cca57a832547a916bd87ba44f3d25b6209/masonry/src/doc/01_creating_app.md) is broken.